e2e/pages/events/manageHearingsEvent.page.js

```diff
@@ -22,13 +22,16 @@ module.exports = {
     hearingType: {
       caseManagement: '#hearingType-CASE_MANAGEMENT',
     },
+    hearingDuration: {
+      dateTime: '#hearingDuration-DATE_TIME',
+    },
     hearingVenue: '#hearingVenue',
     usePreviousHearingVenue: '#previousHearingVenue_usePreviousVenue_Yes',
     dontUsePreviousHearingVenue: '#previousHearingVenue_usePreviousVenue_No',
     newVenue: '#previousHearingVenue_newVenue',
     newVenueCustomAddress: '#previousHearingVenue_newVenueCustomAddress_newVenueCustomAddress',
     startDate: '#hearingStartDate',
-    endDate: '#hearingEndDate',
+    endDate: '#hearingEndDateTime',
```

Comment:
Field hearingEndDateTime will be holding date time selected in the UI. While existing hearingEndDate will be a derived field if days/ hour:minutes is selected or it will be set to hearingEndDateTime so there is minimum impact.